client/modules/workspace/src/JobsDetailModal/JobsDetailModal.tsx

```diff
@@ -1,16 +1,123 @@
 import React, { useState, useEffect } from 'react';
-import { Modal, Layout, Button } from 'antd';
+import { Collapse, Modal, Layout, Button } from 'antd';
 import { useNavigate } from 'react-router-dom';
-import { useGetJobs, TTapisJob } from '@client/hooks';
+import { useGetApps, useGetJobs, TAppResponse, TTapisJob } from '@client/hooks';
 import styles from './JobsDetailModal.module.css';
-import { getStatusText, isOutputState, isTerminalState } from '../utils/jobs';
+import {
+  getOutputPath,
+  getStatusText,
+  isOutputState,
+  isTerminalState,
+  getJobDisplayInformation,
+  TJobDisplayInputOrParameter,
+  TJobDisplayInfo,
+} from '../utils/jobs';
 import { formatDateTimeFromValue } from '../utils/timeFormat';
 import { JobActionButton } from '../JobsListing/JobsListing';
+import { DescriptionList } from '../DescriptionList/DescriptionList';
```

Comment:
Not to discount importing this, but what are your thoughts on using this from Core vs the component provided by Ant which we use in the `<AppsSubmissionDetails />` component?
https://ant.design/components/descriptions

Reply:
good point, I looked at fully reusing that component, but did not go into DescriptionList. I'll use ant component and update the PR.

Reply:
Switched to antd component and tested. A screenshot

<img width="1020" alt="Screenshot 2024-07-09 at 3 07 09 PM" src="https://github.com/DesignSafe-CI/portal/assets/2568355/d35103e1-3afe-4496-9ddb-8e0337c020a2">